UnicornRender/video/source/CameraProjection.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/video/CameraProjection.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+
+CameraProjection::CameraProjection(system::Window* window, glm::mat4& cameraProj) :
```

Comment:
If this constructor does not handles nullptr window, maybe take it by the reference?